src/ui/qgsfieldcalculatorbase.ui

```diff
@@ -186,6 +189,9 @@
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="mOutputFieldWidthLabel">
+        <property name="toolTip">
+         <string>Length of complete output. For example 123,456 means 6 as field length. Includes digits after the decimal places for decimal field types.</string>
```

Comment:
It's not, I keep re-learning about that fact :)

Reply:
How about putting the example at the end?
```suggestion
         <string>Length of complete output. Includes digits after the decimal places for decimal field types. For example 123,456 means 6 as field length.</string>
```

Reply:
I've always thought and repeated that the decimal was also counted, as it was indicated at https://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/attribute_table.html#using-the-field-calculator (just after the screenshot)

Reply:
@DelazJ  I wonder how useful that example is anyway? I think it'd be more useful to have a decimal based example, instead of the integer based one which is kind of obvious anyway and also confusing due to the inclusion of the thousands separator.

Reply:
@nyalldawson you mean as a tooltip for the label, right? I did not see the ui file before my comment, thus it was more about the text structure than its location. Indeed, no need for the label, the spinbox's should be enough.

While we're at it, any idea what the length field means when creating an integer field? (and i guess the tooltip will appear here)
![image](https://user-images.githubusercontent.com/7983394/53316521-8c4d4680-38c8-11e9-9bb7-3cd23518a01a.png)